docs/node-operators/how-to/upgrade-network.md

```diff
@@ -7,57 +7,42 @@ hide_title: false
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-The protocol upgrade feature allows the nodes running a network to automatically update to the latest version of the Vega protocol, without requiring manual intervention. This can be particularly useful in situations where the protocol is critical to the functioning of the system and any downtime or disruption needs to be minimised.
+The Protocol Upgrade feature allows the Node Operators synchronously update nodes to the next version of the Vega Protocol.
 
-Protocol upgrades can be done using Vega Visor, a process runner that manages the the processes of a Vega validator node and a data node. Visor starts and stops node processes, tells the node when to take a snapshot, and coordinates upgrades to the protocol.
+The Protocol Upgrade involves three main parts: 
+1. Submit a transaction to initiate the upgrade on a specific block (validators only)
+2. Prepare a Node for Protocol Upgrade
+3. Execute an upgrade at the agreed block
 
-Alternatively, a validator node operator can manually upgrade the protocol at an agreed block height, for an agreed Vega software version.
+Smooth execution of the Protocol Upgrade is critical to the Vega Network, and any downtime or disruption must be minimised. Using Vega Visor is recommended and will automatically prepare and execute a protocol upgrade. Alternatively, the Node Operator can manually perform an Upgrade at an agreed block height for an agreed Vega software version.
 
-A protocol upgrade involves two main parts: 
-* Submitting a transaction to initiate the upgrade on a specific block
-* Using a process manager, Vega Visor, to coordinate the rollout of the upgrade, or manually upgrading at the agreed block height
+To help with Protocol Upgrades, the data node provides useful information:
+* to view submitted proposals use **[protocol upgrade proposals](../../api/rest/data-v2/trading-data-service-list-protocol-upgrade-proposals.api.mdx) endpoint**,
+* to view if the network is ready for upgrade use **[protocol upgrade status](../../api/rest/data-v2/trading-data-service-get-protocol-upgrade-status.api.mdx) endpoint**.
 
-## Using Visor to coordinate upgrades
-If an upgrade proposal is approved (more than 2/3 of consensus validators have voted on the proposal), those using Visor can rely on it to coordinate the rollout of the upgrade across all the nodes on the chosen block height. 
 
-This includes stopping the currently running nodes, ensuring that the new binaries are the correct software version, and starting new nodes. Visor also includes features such as restart capability, which allows Visor to retry the start-up several times before failing. 
+## 1. Submit a transaction to initiate the upgrade on a specific block (validators only)
```

Comment:
```suggestion
## 1. Submit a transaction to initiate the upgrade on a specific block

This step can only be done by consensus validators.
```
is it true that it's only consensus, not standby? I've added it in but would like confirmation.